frontend/src/bundles/common/components/chip/chip.tsx

```diff
@@ -0,0 +1,25 @@
+import { Close } from '@mui/icons-material';
+import { Chip as MUIChip } from '@mui/material';
+
+import { getValidClassNames } from '../../helpers/helpers.js';
```

Comment:
Don't use relative path in this case use absolute path like '~/bundles/common/helpers/helpers.js'